content/integrations/java.md

```diff
@@ -336,8 +336,10 @@ instances:
   5. Restart the agent: `sudo /etc/init.d/datadog-agent`
 
 
+## Further Reading
 
-
-
-
-
+* [I Have a Matching Bean for my JMX integration but nothing on Collect !](https://help.datadoghq.com/hc/en-us/articles/218277463-I-Have-a-Matching-Bean-for-my-JMX-integration-but-nothing-on-Collect-)
+* [View jmx data in jConsole and set up your jmx.yaml to collect them](https://help.datadoghq.com/hc/en-us/articles/207525586-View-jmx-data-in-jConsole-and-set-up-your-jmx-yaml-to-collect-them)
```

Comment:
I like consistency, so I'd say s/jmx/JMX/ as it's capitalized everywhere else.

Reply:
👍